controller/pagecontroller.php

```diff
@@ -80,9 +80,17 @@ public function __construct(
 	 */
 	public function index() {
 		$appName = $this->appName;
+		$config = \OC::$server->getConfig();
```

Comment:
IConfig is already injected and if it's not the right one, then you need to inject it from application.php instead of using `\OC`.